tests/5-challenge.spec.js

```diff
@@ -0,0 +1,36 @@
+const { readFileSync } = require('fs');
+const { Sequelize } = require('sequelize');
+const Importer = require('mysql-import');
+const { dbCredentials, sequelizeConfig, importerConfig } = require('./config')
+
+
+describe('5 - Crie uma `QUERY` que atualiza todas as pessoas que se cadastraram nos dias 2018-01-16, 2019-09-02 ou 2022-06-01 para 2022-12-31.', function () {
+  let sequelize;
+
+  beforeAll(async () => {
+    const importer = new Importer(importerConfig);
+
+    await importer.import('./tests/utils/trybe-hotel-test.sql');
+
+    importer.disconnect();
+
+    sequelize = new Sequelize('trybe_hotel_test', dbCredentials.user, dbCredentials.password, sequelizeConfig)
+  });
+
+  afterAll(async () => {
+    sequelize.close();
+  });
+
+  it('Se apenas três linhas são atualizadas.', async () => {
```

Comment:
Para quem for ler o log do teste, fica meio estranho esse `se`. Vou deixar a sugestão abaixo tanto aqui como no README. Além disso, uma boa prática para descrição de mensagens de teste é ter algum verbo para indicar o comportamento esperado. Você seguiu essa prática nos demais, mas aqui deu uma rápida desviada desse padrão.

```suggestion
  it('Deve atualizar três linhas.', async () => {
```